content/judgment.tex

```diff
@@ -55,14 +61,4 @@ \subsection{Referee -- Team communication}
 
 \subsection{Referees During the Match}
 
-The head referee and the assistant referees should wear clothing and socks \emph{of black or dark blue color} (blue jeans are acceptable) and avoid reserved colors for the ball, the goals, and player markings in their clothing. They may enter the field in particular situations, \eg, to remove a robot when applying a penalty. They should avoid interfering with the robots as much as possible.
-
-\subsection{A Remark on Artificial Landmarks}
-\label{sec:judgment:landmarks}
-
-The head referee may decide at any point before or during a game to relocate any objects around the field, or direct persons to another position around the field.
-
-The intent of using same-colored goals is to remove artificial landmarks.
-Robots should be able to localize with the SPL field and its ``normal'' surroundings.
-Introducing new team-specific artificial landmarks is against the spirit and intention of the league's progress.
-The application of this rule needs to be well considered and should be reserved for situations which seem constructed by one team or another, but will ultimately be the head referee's decision alone.
+The head referee and the assistant referees should wear socks \emph{of black or dark blue color} (blue jeans are acceptable) and avoid reserved colors for the ball, the goals, and player markings in their clothing. They may enter the field in particular situations, \eg, to remove a robot when applying a penalty. They should avoid interfering with the robots as much as possible.
```

Comment:
I support the idea that the trousers should not be as restricted as they used to be (-> "(blue jeans are acceptable)" should be removed, too). I would replace the "avoid reserved colors" part, too (the only left "reserved colors" are white and green).

Reply:
Done